packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -662,7 +720,8 @@ export class TogglingSyntaxConditionalsTest extends TogglingConditionalsTest {
     this.assertText('T1F2');
   }
 
-  ['@test it tests for `isTruthy` on the context if available']() {
+  // Marking as @htmlbars since this no longer relevant for glimmer as `this` refers to the component, which is never a proxy
+  ['@htmlbars it tests for `isTruthy` on the context if available']() {
```

Comment:
ditto, do we have another shared/generic test that tests changing the content of the proxy?